gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/testing/LocalDatastoreHelperTest.java

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
-package com.google.gcloud.datastore;
+package com.google.gcloud.datastore.testing;
 
 import static org.junit.Assert.assertSame;
 import static org.junit.Assert.assertTrue;
 
 import com.google.gcloud.AuthCredentials;
+import com.google.gcloud.datastore.DatastoreOptions;
 import com.google.gcloud.datastore.testing.LocalDatastoreHelper;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [No need to import a type that lives in the same package](https://www.codacy.com/app/mziccard/gcloud-java/file/2386656939/issues/source?bid=3123103&fileBranchId=3228105#l24)